docs/telegram-usage.md

```diff
@@ -179,6 +180,21 @@ Use `/reload_conf` to reset the blacklist.
 > Using blacklist `StaticPairList` with 2 pairs  
 >`DODGE/BTC`, `HOT/BTC`.
 
+### /edge
+
+Shows pairs accepted by pais along with their corresponding winrate, expectancy and stoploss values.
```

Comment:
"pairs accepted by pais"  ? i assume it should be "pairs" twice - but i'm not sure it makes much sense then either...

Reply:
lol, was drunk I guess !!